Text/Shakespeare/Base.hs

```diff
@@ -283,11 +284,16 @@ readUtf8File fp = do
 --
 -- @since 2.0.19
 readFileQ :: FilePath -> Q String
-readFileQ fp = qRunIO (readUtf8FileString fp)
+readFileQ rawFp = do
+  fp <- makeRelativeToProject rawFp
+  qRunIO (readUtf8FileString fp)
 
 -- | Embed file's content, converting newlines
 -- and track file via ghc dependencies, recompiling on changes
 --
 -- @since 2.0.19
 readFileRecompileQ :: FilePath -> Q String
-readFileRecompileQ fp = addDependentFile fp >> readFileQ fp
+readFileRecompileQ rawFp = do
+  fp <- makeRelativeToProject rawFp
+  addDependentFile fp
+  qRunIO (readUtf8FileString fp)
```

Comment:
I couldn't figure out a nice way to unify these. I didn't want to make `readUtf8FileString` run in `Q`, since it didn't seem concerned with being a TH function.